docker-clean.sh

```diff
@@ -1,7 +1,16 @@
 #!/usr/bin/env bash
 #
-# Cleans everything docker can leave behind.  NOT to be used in a production
-# environment.  You should probably be a little less brute-force with your
-# cleaning if you're not just developing stuff.
-docker rm $(docker ps -qa)
-docker rmi $(docker images | grep "<none>" | awk '{print $3}')
+# Stop and remove the docker containers necessary for open ONI
+# except for the persistent data containers
+
+echo "stopping ..."
+docker stop openoni-dev
```

Comment:
we just had a chat in the slack channel and we've decided on multiple lines to aid debugging, should one of the containers fail to stop / rm